restservice/src/main/java/org/itevents/controller/ServiceExceptionHandler.java

```diff
@@ -38,11 +41,13 @@
 
     @ExceptionHandler(ActionAlreadyDoneServiceException.class)
     public ResponseEntity<String> handleActionAlreadyDoneServiceException(ActionAlreadyDoneServiceException ex) {
+        LOGGER.error(ex.getMessage());
         return new ResponseEntity<>(ex.getMessage(), HttpStatus.CONFLICT);
     }
 
     @ExceptionHandler(ConstraintViolationException.class)
-    public ResponseEntity<String>  handleConstrainViolationExceptions(ConstraintViolationException ex) {
+    public ResponseEntity<String>  handleConstrainViolationException(ConstraintViolationException ex) {
+        LOGGER.error(ex.getMessage());
         return new ResponseEntity<>(ex.getMessage(), HttpStatus.BAD_REQUEST);
```

Comment:
@IgorMaksymov I think that it is insecure to pass message of `ConstraintViolationException` to `ResponseEntity`. Create custom message.